src/rpc/protocol.h

```diff
@@ -63,6 +63,9 @@ enum RPCErrorCode
     RPC_CLIENT_INVALID_IP_OR_SUBNET = -30, //!< Invalid IP/Subnet
     RPC_CLIENT_P2P_DISABLED         = -31, //!< No valid connection manager instance found
 
+    //! Chain errors
```

Comment:
nit: I think this change dissociate further chain from mempool, so may qualify better as a mempool error.

Reply:
I think it is fine to keep as is, as mempool and chain are closely related. Also, I'd rather not invalidate ACKs over this.